internal/feeder/bastion/bastion_feeder.go

```diff
@@ -109,15 +142,17 @@ func (a *addHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if !ok {
 		klog.V(1).Infof("unknown log: %v", logID)
 		w.WriteHeader(http.StatusNotFound)
+		counterBastionIncomingResponse.Inc(bastionID, "unknown", strconv.Itoa(http.StatusNotFound))
```

Comment:
The strategy for incrementing this counter isn't clear to me, and I think it'll make coming up with a useful metric more difficult than it needs to be. The docs say "Number of _successful_ bastion mediated requests for a log", but you've added status as a dimension, and incrementing it here, which indicates it isn't just successful ones. But you haven't incremented it on the bad request responses above.

I would suggest just leaving it to successes, OR increment it by all explicit returns from this method. Then any different between request and response will be crashes / deadlocks etc, which shouldn't happen.

Reply:
The description was out of date (originally it was just successes, but counting by response code is was very useful in figuring out why things weren't quite working properly).